.circleci/config.yml

```diff
@@ -97,24 +97,45 @@ jobs:
   #   environment:
   #     API_KEY: somefakeapikey
   #     APPLICATION_KEY: somefakeapplicationkey
+  #     CHEF_VERSION: '14.0'
   #     RUBY_VERSION: '2.3.8'
-  #     BUNDLE_GEMFILE: gemfiles/chef_14.gemfile
 
   specs-ruby24-chef-140:
     <<: *specs
     environment:
       API_KEY: somefakeapikey
       APPLICATION_KEY: somefakeapplicationkey
+      CHEF_VERSION: '14.0'
       RUBY_VERSION: '2.4.9'
-      BUNDLE_GEMFILE: gemfiles/chef_14.gemfile
 
   specs-ruby25-chef-140:
     <<: *specs
     environment:
       API_KEY: somefakeapikey
       APPLICATION_KEY: somefakeapplicationkey
+      CHEF_VERSION: '14.0'
       RUBY_VERSION: '2.5.1'
-      BUNDLE_GEMFILE: gemfiles/chef_14.gemfile
+
+  verify-gemfile.lock-dependencies: &specs
```

Comment:
I think this `&specs` wasn't meant to be here.

Reply:
Good catch!